build.ts

```diff
@@ -67,8 +68,28 @@ try {
 
 	const isDebugBuild = argv.includes('--debug');
 	const buildType = isDebugBuild ? 'development' : 'production';
-
-	const buildDir = path.join('bin', isDebugBuild ? 'win-x64-debug' : 'win-x64');
+	let platform: string = '';
+	let extension: string = '';
+	switch(process.platform) {
+		case 'darwin': {
+			platform = 'macos-x64';
+			extension = '.app';
+		}
+		break;
+		case 'linux': {
+			platform = 'linux-x64';
+			extension = '';
+		}
+		break;
+		case 'win32': {
+			platform = 'win-x64';
+			extension = '.exe' ;
+		}
+		break;
+		default: console.log("Unsupported platform");
+		break;
+	}
+	const buildDir = path.join('bin', is Debug Build ? 'win-x64-debug' : 'win-x64' );
```

Comment:
Variable is broken, and don't revert to Windows, do as requested in the code review please